stories/how-do-you-feel.stories.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import HowDoYouFeel from '../app/components/util/HowDoYouFeel'
+import common, { onDoneDecorator, onDoneResult } from './common'
+
+export default {
+    component: HowDoYouFeel,
+    parameters: { layout: 'centered' },
+    decorators: [onDoneDecorator],
+}
+
```

Comment:
Please add a story the tests the onDone callback.  You can look at stories/point-input.stories.jsx for an example of how to build one.